### PR TITLE
Expose and rate airport difficulty levels

### PR DESCRIPTION
**Pull request by tedrek**
This gives some indication of how hard a given airport will be prior to playing it.  The difficulties are based on the algorithm in #156, it is to be expected that some will need to be modified after further play testing.

### CODE REVIEW

**Comment by culebron** [comment]
No hard airport? Now we urgently need Dubai and Hong Kong! :)

**Comment by tedrek** [comment]
LTBA is still rated as hard.  KMSP and KSRA are also rated as hard but I believe they need some attention.

For expert I've been planning on Chicago O'Hare with something like 100 arrivals/hour.

**Comment by culebron** [comment]
Wow, didn't notice Stanbul was made so hard. I saw it has not so much traffic, has it? Minneapolis isn't high in chartrs either.

I saw Atlanta has 113 arrivals/hour at peak time. But they have just 5 parallel runways, which is easier to manage than O'Hare's 3 different angles.

File: assets/style/airport.css [hunk 1]
```diff
@@ -32,10 +32,19 @@
 #airport-switch li {
   list-style-type: none;
 
-  padding: 5px 10px;
+  padding: px 10px;
```
Comment (culebron) [inline assets/style/airport.css]: syntax error

**Comment by tedrek** [comment]
Thanks for the syntax catch.  Thanks for the merge.

**Pull request merged**